roles/confluent.kafka_broker/tasks/rbac.yml

```diff
@@ -1,11 +1,5 @@
 ---
-- name: Get Principal used on the Inter Broker Listener
-  include_tasks: set_principal.yml
-  vars:
-    listener: "{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]}}"
-  when: install_pattern == 'parallel'
```

Comment:
Is it necessary to remove the parralel vs serial condition? I think I kept the parallel option to make the set_principal tasks run faster in a parallel deployment

Reply:
The issue I found with the parallel loop was that if there is a failed broker it is first placed into the  kafka_broker_parallel group, and available members into kafka_broker_serial. The play then targets kafka_broker_parallel, with only the single member. At this point, include_tasks without the loop only executes for the single broker and doesn't collect the principal data from all the brokers as expected.

As for speed optimisation - I added run_once to the loop, which I think addresses the same concern.

Reply:
Thanks for the explanation, i missed the run_once!